bot.go

```diff
@@ -36,6 +36,18 @@ func New() {
 	<-sc
 	return
 }
+func getNumOptions() []string {
+	arr := []string{"1️⃣", "2️⃣", "3️⃣", "4️⃣", "5️⃣", "6️⃣", "7️⃣", "8️⃣", "9️⃣"}
+	return arr
+}
```

Comment:
数字を引数にとって対応する絵文字を返すような関数にしてしまってもいいかも
```suggestion
func getNumEmoji(i int) string {
	if i < 1 {
		return "❓"
	}
	// 対応する絵文字がない場合はその値をそのまま返す
	if i > 9 {
		return strconv.Itoa(i)
	}
	arr := []string{"1️⃣", "2️⃣", "3️⃣", "4️⃣", "5️⃣", "6️⃣", "7️⃣", "8️⃣", "9️⃣"}
	return arr[i-1]
}
```

Reply:
ありがとうございます。
正直getNumEmoji関数あったら便利なんでしょうけど、現状の使い道が思いつきません。
タイトル数を取得して、その数に対応する絵文字リストを返し、リストの中からその文字列を探すための数が減る。
(トップレベルで定義したときの 9個と探索数はあんまり変わらない気がしてしまいます。)
なのでトップレベルで定義してgetNumEmojiは必要に応じて実装する、とかでいいのかなと思います。